mne/viz/evoked.py

```diff
@@ -1345,14 +1345,17 @@ def _ci(arr, ci):
     Aux function for plot_compare_evokeds.
     """
     from scipy import stats
+
     mean, sigma = arr.mean(0), stats.sem(arr, 0)
     # This is highly convoluted to support 17th century Scipy
     # XXX Fix when Scipy 0.12 support is dropped!
     # then it becomes just:
     # return stats.t.interval(ci, loc=mean, scale=sigma, df=arr.shape[0])
-    return np.asarray([stats.t.interval(ci, arr.shape[0],
-                       loc=mean_, scale=sigma_)
-                       for mean_, sigma_ in zip(mean, sigma)]).T
+    ci_band = np.asarray([stats.t.interval(
+            ci, arr.shape[0], loc=mean_, scale=sigma_)
+            for mean_, sigma_ in zip(mean, sigma)]).T
+
+    return (ci_band)
```

Comment:
I'm not sure this is an improvement ..? Especially considering it'll get replaced with a 1-liner once we update the minimum scipy.